### DISCUSSION TITLE
I need help, Phaser 3 Blackscreen

### DISCUSSION THREAD

**Discussion by AureliusEdran97** [photonstorm/phaser#6132] [General]
i'm very new in coding and i look a lot of tutorials and i follow even the step of phaser 3 official website for my first videogame, but no metter what i write in visual studio code, the only result in a browser web page with blackscreen, i look for error in the tab but there aren't, i don't know what to do, someone can help?
-->

**Comment by photonstorm** [comment]
There are too many unknowns to be able to really help. Some things to start with:

1) What web server as you using? You need something installed locally to serve your game.
2) Does Phaser even load? Is the whole page black, or just the canvas? Try adding some HTML onto the page, does it appear?
3) How are you opening your game? You should be going to a URL on your local server.
